### DISCUSSION TITLE
Active Live Streams

### DISCUSSION THREAD

**Discussion by brixxnatt** [ant-media/Ant-Media-Server#6469] [General]
yes  , Active Live Streams show for example 5 ,   "Active Live Streams 5"  , it will be very helpfull if  on mouse hover can get the list of the LIVE channels names or stream names  now  . 
thanks

**Comment by Mohit-3196** [comment]
Hi @brixxnatt,
How are you?

Sorry, this is not clear to me.
Can you please describe what you would like to achieve in detail?

Thank you.
